app/models/framework/definition/ast/semantic_checker.rb

```diff
@@ -77,7 +77,8 @@ def raise_when_lookup_reference_invalid(field)
         end
 
         def raise_when_management_field_invalid(info)
-          referenced_field_name = [info.dig(:column_based, :column_name), info.dig(:flat_rate, :column)].compact.first(&:present?)
+          # TODO: handle multiple entries in :column_names
```

Comment:
Is this still a TODO? Is that not what this PR does?

Reply:
No. This is to make sure an error is raised if you and reference a column that doesn't exist in your management charge calculation (and would therefore always fallback).

Looks like I forgot to do this part!